src/components/NurseryWithdrawals/WithdrawalDetails/sections/OutplantWithdrawalTable.tsx

```diff
@@ -23,49 +36,93 @@ export default function OutplantWithdrawalTable({
+    batchId: number;
+    speciesId: number;
+    toSubzone: string;
+  };
+
   useEffect(() => {
-    // get list of distinct species
-    const speciesList =
-      delivery?.plantings?.reduce<number[]>(
-        (acc, pl) => (acc.includes(pl.speciesId) ? acc : [...acc, pl.speciesId]),
-        []
-      ) ?? [];
-    const speciesSubzoneMap: Record<number, Record<number, number>> = {};
-    const rows: { [p: string]: unknown }[] = [];
-    for (const sp of speciesList) {
-      // for each species add the number of plants in each subzone for delivery type plantings
-      speciesSubzoneMap[sp] = {};
-      delivery?.plantings
-        ?.filter((pl) => pl.speciesId === sp && pl.type === 'Delivery')
-        .forEach((pl) => {
-          const subzone = pl.plantingSubzoneId ?? -1;
-          if (!speciesSubzoneMap[sp][subzone]) {
-            speciesSubzoneMap[sp][subzone] = pl.numPlants;
-            return;
-          }
-          speciesSubzoneMap[sp][subzone] += pl.numPlants;
-        });
+    if (batches) {
+      const batchToSpeciesMap = batchToSpecies(batches);
+
+      // get list of distinct species
+      const speciesList =
+        delivery?.plantings?.reduce<number[]>(
+          (acc, pl) => (acc.includes(pl.speciesId) ? acc : [...acc, pl.speciesId]),
+          []
+        ) ?? [];
+      const speciesSubzoneMap: Record<number, Record<number, number>> = {};
+      const rows: { [p: string]: unknown }[] = [];
+      for (const sp of speciesList) {
+        // for each species add the number of plants in each subzone for delivery type plantings
+        speciesSubzoneMap[sp] = {};
+        delivery?.plantings
+          ?.filter((pl) => pl.speciesId === sp && pl.type === 'Delivery')
+          .forEach((pl) => {
+            const subzone = pl.plantingSubzoneId ?? -1;
+            if (!speciesSubzoneMap[sp][subzone]) {
+              speciesSubzoneMap[sp][subzone] = pl.numPlants;
+              return;
+            }
+            speciesSubzoneMap[sp][subzone] += pl.numPlants;
+          });
 
-      // transform into table rows
-      for (const subzoneKey of Object.keys(speciesSubzoneMap[sp])) {
-        const subzone = Number(subzoneKey);
-        rows.push({
-          species: species?.find((x) => x?.id === sp)?.scientificName ?? '',
-          to_subzone: subzone > -1 ? subzoneNames[subzone] ?? subzone?.toString() : '',
-          quantity: numericFormatter.format(speciesSubzoneMap[sp][subzone]),
+        for (const subzoneKey of Object.keys(speciesSubzoneMap[sp])) {
+          const subzone = Number(subzoneKey);
+          rows.push({
+            species: species?.find((x) => x?.id === sp)?.scientificName ?? '',
+            to_subzone: subzone > -1 ? subzoneNames[subzone] ?? subzone?.toString() : '',
+            quantity: numericFormatter.format(speciesSubzoneMap[sp][subzone]),
+          });
+        }
+      }
+
+      const batchesMap: BatchesRow[] = [];
+
+      if (Object.keys(batchToSpeciesMap).length > 0) {
```

Comment:
maybe we can add a nurseryV2 check here as well so we aren't doing this in production until released